src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.xml

```diff
@@ -80,10 +80,10 @@
         <service id="Symfony\Component\Routing\RequestContextAwareInterface" alias="router" />
 
         <service id="router.request_context" class="Symfony\Component\Routing\RequestContext">
-            <argument></argument> <!-- base_url -->
-            <argument>GET</argument>
-            <argument></argument> <!-- host -->
-            <argument></argument> <!-- scheme -->
+            <factory class="Symfony\Component\Routing\RequestContext" method="fromBaseUri" />
+            <argument>%router.request_context.base_url%</argument>
```

Comment:
This parameter name too

Reply:
This parameter already exists, we cannot rename it.